src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -138,13 +139,13 @@ export class WizardModal extends Modal {
 		this._wizard.onPageAdded(page => {
 			this.registerPage(page);
 			this.updatePageNumbers();
-			this.showPage(this._wizard.currentPage, false, false, false).catch(err => onUnexpectedError(err));
+			this.showPage(this._wizard.currentPage, -1, false, false, false).catch(err => onUnexpectedError(err));
```

Comment:
I'd just add a new parameter called sendTelemetry or something to determine whether you send the event or not.

Reply:
This was specifically used for prevIndex, with the new change this won't be required, hence altered the changes to its original state.